test-suite/test-sources.sh

```diff
@@ -229,7 +229,8 @@ main() {
     fi
     echo "Starting point: $STARTING_POINT (`git rev-parse $STARTING_POINT`)"
 
-    checks="$@"
+    local checks="$@"
```

Comment:
This out of scope polishing change reduces unwanted differences between the two test-suite scripts. There is more about this code duplication in the PR that introduced it at https://github.com/squid-cache/squid/pull/1160/files#r986398359